docs/content/preview/faq/yugabyte-platform.md

```diff
@@ -123,3 +138,88 @@ Yes, you can control what YugabyteDB Anywhere is spinning up. For example:
 - You can choose dedicated IOPs EBS drives on AWS and specify the number of dedicated IOPS you need.
 
 YugabyteDB Anywhere also allows creating these machines out of band and importing them as an on-premises install.
+
+## Node agent
+
+### What is a node agent?
+
+Node agent is an RPC service running on a YugabyteDB node, and is used to manage communication between YugabyteDB Anywhere and the database node. It includes the following functions:
+
+- Invoke shell commands directly on the remote host like running commands over SSH. Similar to SSH, the agent also does shell-login such that any previous command modifying the environment in the resource files (for example, `~/.bashrc`) gets reflected in the subsequent command.
+- Invoke procedures or methods on the node agent.
+- For on-premesis (manual provisioned nodes), node agent functions as a utility to run preflight checks, and add a node instance.
```

Comment:
node agent **also** functions as these functions are in addition to the communication agent (in the intro line).